_posts/2019-06-25-CRANsearcher-package-discovery.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: CRANsearcher - R package discovery made simple
+author: Becca Krouse and Agustin Calatroni
+---
+
+It's a familiar story to many of us: we're working along and find that we need something very specific.  It could be a function or macro, or maybe an analysis technique.  It's also probably going to be quite difficult to create from scratch.  So we wonder, maybe this has been done before?  
+
+As data scientists, exploratory analyses performed using emerging techniques are a regular part of the job.  We are pushing the limits of research and we are constantly trying to discover and compare existing methods. We find ourselves in this position often - taking inventory of our analytical toolbox and trying to avoid reinventing the wheel.  Much of our data science work at Rho is performed using R, which offers a vast and rapidly growing ecosystem of [packages](https://cran.r-project.org/web/packages/available_packages_by_name.html).  Indeed, R packages extend from visualization to Bayesian inference, and from spatial analyses to pharmacokinetics. There is probably not an area of quantitative research that isn't represented by at least one R package.  R's great strength, however, can also be a weakness.  To answer the question of "has someone already created a tool for this" or "what is the best way to do X", we find ourselves navigating package websites, subscribing to blogs, and doing lots of googling. Unfortunately, this is also a diversion from our usual analysis workflow and is not always productive.   
```

Comment:
```suggestion
As data scientists, exploratory analyses performed using emerging techniques are a regular part of the job.  We are pushing the limits of research and  constantly trying to discover and compare existing methods. We often find ourselves taking inventory of our analytical toolbox and trying to avoid reinventing the wheel.  Much of our data science work at Rho is performed using R, which offers a vast and rapidly growing ecosystem of [packages](https://cran.r-project.org/web/packages/available_packages_by_name.html).  Indeed, R packages extend from visualization to Bayesian inference, and from spatial analyses to pharmacokinetics. There is probably not an area of quantitative research that isn't represented by at least one R package.  R's great strength, however, can also be a weakness.  To answer the question of "has someone already created a tool for this" or "what is the best way to do X", we find ourselves navigating package websites, subscribing to blogs, and doing lots of googling. Unfortunately, this is also a diversion from our usual analysis workflow and is not always productive.   
```